src/olympia/addons/models.py

```diff
@@ -654,8 +656,10 @@ def force_disable(self):
         # https://github.com/mozilla/addons-server/issues/13194
         self.disable_all_files()
 
-    def force_enable(self):
-        activity.log_create(amo.LOG.FORCE_ENABLE, self)
+    def force_enable(self, user=None):
+        activity.log_create(
+            amo.LOG.FORCE_ENABLE, self, **({'user': user} if user else {})
+        )
```

Comment:
As above, we're missing at least `details` in there, which would record the comment if any.

Reply:
I resolved both of these comments by skipping the logging in the Addon function, and relying on the reviewer tools `log_action`